manager/controllers/filter.go

```diff
@@ -2,12 +2,15 @@ package controllers
 
 import (
 	"app/base/database"
+	"app/base/utils"
 	"fmt"
 	"github.com/pkg/errors"
 	"gorm.io/gorm"
 	"strings"
 )
 
+var otherAdvisoryTypes = strings.Split(utils.Getenv("OTHER_ADVISORY_TYPES", "unspecified,unknown"), ",")
+
```

Comment:
Maybe we could get this list directly from database at startup. E.g.
`select name from advisory_type where name not in ('enhancement', 'bugfix', 'security');`

Reply:
Ok, updated.